ipcore/avalon_wrapper.py

```diff
@@ -70,6 +70,12 @@ def generate_avalon_wrapper(registers, audio_in, audio_out, entity_name, working
         Port(PortDir.In, Signal("avalon_slave_write")),
         Port(PortDir.In, Signal("avalon_slave_writedata", 32))
     ]
+
+    if(audio_in.dual):
```

Comment:
What does dual mean?

Reply:
Dual inputs. Pretty good chance it will actually end up getting removed but its been added as a cheap way to sort of support 2 audio inputs at once because we need noise to be passed in for the auditory nerve model